engine/runtime/src/test/scala/org/enso/interpreter/test/semantic/ImportsTest.scala

```diff
@@ -77,4 +77,13 @@ class ImportsTest extends PackageTest {
     )) should have message "Compilation aborted due to errors."
     consumeOut should contain("Export statements form a cycle:")
   }
+
+  "Import statements" should "should allow for importing submodules" in {
```

Comment:
I remember discussions on what to do when we both have stuff like module `A/B.enso` defining `type C` and a module `A/B/C.enso`.

I don't see any tests checking how this conflict-case behaves. Can we add those? (Or did I miss something?)

Reply:
Yeap, was going to add it. Will need to add some additional check to produce a reasonable message

Reply:
@radeusgd done